tests/recipient_validation/test_phone_number.py

```diff
@@ -93,7 +83,7 @@
                 ),
             ),
             (
-                InvalidPhoneError.ERROR_MESSAGES[InvalidPhoneError.Codes.TOO_SHORT],
+                InvalidPhoneError.ERROR_MESSAGES[InvalidPhoneError.Codes.INVALID_NUMBER],
```

Comment:
The phonenumbers library throws different errors when trying to run `phonenumbers.is_valid_number(number)` against a number that has been successfully parsed as "possible", but is shown to be invalid when the library tries to validate the number against it's metadata catalogue (it contains a list of regular expressions for each country and matches against them).

This just returns a boolean (it's either valid or it isn't) but doesn't attempt to figure out how or why it didn't match any known regular expression. Without superimposing the old validation code onto this that **does** do a check against the strings for length _etc._ I'm not sure if we can get exactly the same error messages to return.

Reply:
That explanation would do well in a commit message for this change 🙌🏼